app/common/tables.html

```diff
@@ -0,0 +1,22 @@
+<div class="col-sm-8">
+   <center>
+      <h4>View Poll Data</h4>
+      {{OutletCtrl.outletdata}}
```

Comment:
sir removed outlet data this i have used to checked whether data is coming to directive or not from the controller.